helpers/tempesta.py

```diff
@@ -116,12 +116,17 @@ def parse(self, stats):
         self.cl_established_connections = self.parse_option(stats, "Client established connections")
         self.cl_conns_active = self.parse_option(stats, "Client connections active")
         self.cl_rx_bytes = self.parse_option(stats, "Client RX bytes")
-        self.cl_priority_frame_exceeded = self.parse_option(stats, "Client priority frames number exceeded")
+        self.cl_priority_frame_exceeded = self.parse_option(
+            stats, "Client priority frames number exceeded"
+        )
```

Comment:
Do you use other `black` configuration?  We use 100 characters per line. See `tox.ini` in main directory.